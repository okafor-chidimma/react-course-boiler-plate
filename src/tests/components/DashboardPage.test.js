import React from 'react';
import { shallow } from 'enzyme';
import DashBoardPage from '../../components/DashboardPage';

describe('Rendering <DashBoardPage/>', () => {
  it('Renders <DashBoardPage/> correctly', () => {
    const wrapper = shallow(<DashBoardPage/>);
    expect(wrapper).toMatchSnapshot();
  })
})