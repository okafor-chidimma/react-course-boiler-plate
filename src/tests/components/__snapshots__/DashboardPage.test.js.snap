// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering <DashBoardPage/> Renders <DashBoardPage/> correctly 1`] = `
<div>
  Dashboard content page
</div>
`;
